component/Page/Profile/index.tsx

```diff
@@ -154,7 +155,9 @@ export default function ProfilePage(props: PageProps) {
             </Helmet>
             <LayoutContainer backgroundImage="/image/background-profile.svg">
                 <Header fixedPosition={false}/>
-                <div className="flex flex-1-0-0 w-full justify-center z-50">
+                <div className="flex flex-1-0-0 w-full justify-center" style={{
+                    zIndex: 1000000
+                }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in style seems to be an attempt to increase the z-index. However, using a very large number for z-index can lead to unexpected behavior and conflicts with other elements' z-indexes. It's better to use a more reasonable value unless there's a specific need for such a high value.
```diff
-                 <div className="flex flex-1-0-0 w-full justify-center z-50">
+                 <div className="flex flex-1-0-0 w-full justify-center" style={{
+                     zIndex: 1000000
+                 }}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->